extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -569,10 +578,12 @@ describes.realWin(
 
           expect(stateValue).to.equal('accepted');
           expect(stateManagerInfo).to.deep.equal({
-            'consentState': 1,
+            'consentState': CONSENT_ITEM_STATE.ACCEPTED,
```

Comment:
Thanks for updating all the `consentMetadata`. I think it makes sense to keep one or two tests with the value undefined. 
`consentString` should still work without consent_string_type defined because `<amp-consent>` was designed to support any custom consent string format.